### ISSUE TITLE
Rewire Fire: sanity option

### ISSUE THREAD

**Issue by mekkablue** [mekkablue/Glyphs-Scripts#233]
Rewire Fire hangs on glyphs with many paths like this one: ▒

- [ ] Add an option _Skip glyphs with more shapes than: [...]_

**Comment by schriftgestalt** [comment]
You can optimize it quite a bit by comparing the bounds of the paths in the double loop. 
```
    for path1 in layer.paths:
        bounds1 = NSInsetRect(path1.bounds, -0.5, -0.5)
        for path2 in layer.paths:
            bounds2 = path2.bounds
            if not NSIntersectRects(bounds1, bounds2):
                continue
            # do stuff
```

**Comment by mekkablue** [comment]
Thx. I implemented it in commit 3ebba04. For later reference, it is `NSIntersectsRect` not `NSIntersectRects`.
